library/network/test/susefirewalld_services_test.rb

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env rspec
+
+require_relative "test_helper"
+
+Yast.import "SuSEFirewallServices"
+Yast.import "SCR"
+Yast.import "FileUtils"
+
+describe Yast::SuSEFirewallServices do
+
+  before :example do
+    # We shouldn't run this test if firewalld is running
+    if Yast::SuSEFirewallServices.is_a?(Yast::SuSEFirewall2ServicesClass)
+      skip "SuSEfirewall2 backend is not supported by this test" do
```

Comment:
This design of the tests seems unfortunate. If firewalld is stopped, as it is in OBS and in Jenkins, I get
```
# rspec -f d --color ./library/network/test/susefirewall*test.rb
...
91 examples, 0 failures, 46 pending
```
If I run firewalld, some tests will fail. So it seems we are including tests that are not being run and provide a false sense of correctness:
```
# rspec -f d --color ./library/network/test/susefirewall*test.rb
...
91 examples, 14 failures, 45 pending
```